src/corehost/common/pal.h

```diff
@@ -54,12 +54,15 @@
 #if defined(_WIN32)
 #define LIB_PREFIX
 #define MAKE_LIBNAME(NAME) (_X(NAME) _X(".dll"))
+#define BASE_HOST_RID _X("win")
```

Comment:
A thing I dislike about this approach is that a lot of times the runtime assets aren't in the base rid, instead they are in the lowest version of the base rid.

ex.

`win7-x64`
`osx.10.10-x64`

And we will be missing these assets.

Can we use the lowest OS version as the "base rid" for `win` and `osx`?  `linux` is different because we **will** have assets just in `linux-x64`.